pom.xml

```diff
@@ -62,17 +67,21 @@
 
     <opentracing.version>0.33.0</opentracing.version>
     <coveralls-maven-plugin.version>4.3.0</coveralls-maven-plugin.version>
-    <jacoco-maven-plugin.version>0.8.3</jacoco-maven-plugin.version>
+    <jacoco-maven-plugin.version>0.8.6</jacoco-maven-plugin.version>
 
-    <neo4j-java-driver.version>4.0.0</neo4j-java-driver.version>
-    <neo4j-harness.version>3.5.14</neo4j-harness.version>
+    <neo4j-java-driver.version>4.2.0</neo4j-java-driver.version>
+    <neo4j-harness.version>4.2.3</neo4j-harness.version>
+    <reactor.version>3.4.2</reactor.version>
+    <junit.version>4.13.1</junit.version>
+    <awaitility.version>4.0.0</awaitility.version>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>io.opentracing</groupId>
       <artifactId>opentracing-api</artifactId>
       <version>${opentracing.version}</version>
+      <scope>provided</scope>
```

Comment:
why scope is changed?

Reply:
Because we don't want to have a transitive dependency on the OpenTracing API jar. 
Also, `neo4j-java-driver` is declared with `provided` scope for the same reason I believe.
I've checked some other libraries as well: For example, java-reactor: [java-reactor](https://github.com/opentracing-contrib/java-reactor/blob/master/pom.xml#L88)

Reply:
I don't mind changing it back to `compile` if you believe this scope shouldn't be changed

Reply:
yes, please

Reply:
reverted this change back